gp-includes/assets-loader.php

```diff
@@ -17,6 +17,7 @@ function gp_register_default_styles() {
 	// Register our base style.
 	wp_register_style( 'gp-base', $url . '/style' . $suffix, array(), filemtime( $path . 'style' . $suffix ) );
 	wp_register_style( 'gp-jquery-webui-popover', $url . '/jquery.webui-popover' . $suffix, array( 'gp-base' ), filemtime( $path . 'jquery.webui-popover' . $suffix ) );
+	wp_register_style( 'driver-js', $url . '/driver-js' . $suffix, array(), '1.3.0' );
```

Comment:
Please, change the hard-coded version to a one based in the filemtime:

```suggestion
	wp_register_style( 'driver-js', $url . '/driver-js' . $suffix, array(), filemtime( $path . 'driver-js' . $suffix ) );
```